src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -183,6 +195,14 @@ void initializeReference() {
         reference = referenceArguments.getReferenceFile() != null ? ReferenceDataSource.of(referenceArguments.getReferenceFile()) : null;
     }
 
+    private static SeekableByteChannel addPrefetcher(int cloudPrefetchBuffer, SeekableByteChannel x) {
+        try {
+            return new SeekableByteChannelPrefetcher(x, cloudPrefetchBuffer * 1024 * 1024);
+        } catch (IOException ex) {
+            throw new GATKException("Unable to initialize the prefetcher: " + ex);
+        }
+    }
```

Comment:
yes I know, as I said this required a bit more work than expected because doing it this way broke the tests. It's in now.